fineract-accounting/src/main/java/org/apache/fineract/accounting/producttoaccountmapping/domain/ProductToGLAccountMapping.java

```diff
@@ -63,13 +64,14 @@ public class ProductToGLAccountMapping extends AbstractPersistableCustom<Long> {
     @Column(name = "financial_account_type", nullable = true)
     private int financialAccountType;
 
-    @Column(name = "charge_off_reason_id", nullable = true)
-    private Long chargeOffReasonId;
+    @ManyToOne
+    @JoinColumn(name = "charge_off_reason_id", nullable = true)
+    private CodeValue chargeOffReason;
 
     public static ProductToGLAccountMapping createNew(final GLAccount glAccount, final Long productId, final int productType,
-            final int financialAccountType, final Long chargeOffReasonId) {
+            final int financialAccountType, final CodeValue chargeOffReasonId) {
```

Comment:
minor:could you please rename the parameter

Reply:
Nice catch!